src/shared/resources/basic/index.ts

```diff
@@ -4,102 +4,122 @@ interface IBasicFieldType {
   name: BasicField
   value: string
   submitted: boolean
+  answerArray: boolean
```

Comment:
is this a backwards compatible change? the shared folder may not be updated on the clientside, meaning clients may not send over the `answerArray` key in the fields when they try to mutate a form and fail

Reply:
this doesn't change what the clients send us, that's handled by the viewmodels and the `submitForm` function, which are independent of `shared`. the client's behaviour is completely unchanged because this `answerArray` field is only used on the server. so it doesn't matter if the client is updated or not